MovieQuiz/Presentation/MovieQuizPresenter.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  MovieQuizPresenter.swift
+//  MovieQuiz
+//
+//  Created by Alexander Farizanov on 27.12.2022.
+//
+import UIKit
+final class MovieQuizPresenter: QuestionFactoryDelegate {
```

Comment:
между любыми логическими блоками давай оставлять не больше одной пустой строки
```suggestion
import UIKit

final class MovieQuizPresenter: QuestionFactoryDelegate {
```